### PR TITLE
blog: Two months in Servo (December 2023, January 2024)

### PR DESCRIPTION
**Pull request by delan**

### CODE REVIEW

**Comment by sagudev** [comment]
We also update mozangle in december: https://github.com/servo/servo/pull/30546, while this might seem small, this new version mozangle contains new version of ANGLE (before it was from 2019) and complete refactored build scripts.

**Comment by delan** [comment]
> We also update mozangle in december: [servo/servo#30546](https://github.com/servo/servo/pull/30546), while this might seem small, this new version mozangle contains new version of ANGLE (before it was from 2019) and complete refactored build scripts.

Thanks! I didn’t mean to exclude that, it looks like [my script](https://github.com/servo/servo.org/blob/976dda35a1cf553e7717aeb2062367da61f4ab76/tools/list-commits-by-nightly.sh) has a bug where the commit ranges for a nightly can be invalid sometimes, but I didn’t notice because the errors went to stderr (006b3f73864a540f37bf6dfac5e58ccdd3ac95a2).

**Comment by delan** [comment]
2023-12-11 was released from a branch other than main:

```
$ < tools/runs.json jq -r '.workflow_runs[] | select(.updated_at | test("2023-12-1[12]")) | "\(.head_sha)\t\(.updated_at)\t\(.html_url)"'
```
* 48a95b2471c590a0ee1b85710fbb5a37ee8215c5<br>2023-12-12T06:15:50Z<br>https://github.com/servo/servo/actions/runs/7177279988
* 2668a0a43a19643922409e623880558dedfb4b98<br>2023-12-11T05:28:33Z<br>https://github.com/servo/servo/actions/runs/7163021850

5d7995d26473f6f11ca20376d272aed43d60566e fixes that bug!

File: _posts/2024-01-26-two-months-in-servo.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+Several crates have been moved into [our main repo](https://github.com/servo/servo):
+
+* `hyper_serde` ([servo/hyper_serde](https://github.com/servo/hyper_serde)) was moved to `components/hyper_serde` ([@mrobinson](https://github.com/mrobinson), [#30803](https://github.com/servo/servo/pull/30803))
+* `device` ([servo/devices](https://github.com/servo/devices)) was merged into `components/bluetooth` ([@mrobinson](https://github.com/mrobinson), [#30974](https://github.com/servo/servo/pull/30974))
+* `blurmac` ([servo/devices](https://github.com/servo/devices)) was moved to `third_party/blurmac` ([@mrobinson](https://github.com/mrobinson), [#30974](https://github.com/servo/servo/pull/30974))
+
+We’ve also made some dev changes:
+
+- the default branch in [our main repo](https://github.com/servo/servo) is now `main` ([@mrobinson](https://github.com/mrobinson), [@atouchet](https://github.com/atouchet), 23add0c1e5c9c, [#30877](https://github.com/servo/servo/pull/30877))
```
Comment (atbrakhi) [inline _posts/2024-01-26-two-months-in-servo.md:64]: I am not sure if just adding commit id `23add0c1e5c9c` is helpful. maybe we can add the link?

**Review by atbrakhi — APPROVED** [review]
just a small comments otherwise looks great. thanks for working on this!

File: _posts/2024-01-26-two-months-in-servo.md (hunk 1 above)
Comment (delan) [reply _posts/2024-01-26-two-months-in-servo.md]: oops, done in 0bbd3df3cc9feee69c5f5bf0e08dee219f4e840b

**Pull request merged**